File: Python/Product.py
# Saya Alif Faturahman Firdaus (2107377) mengerjakan Latihan Praktikum 2 dalam mata 
# kuliah DPBO untuk keberkahan-Nya maka saya tidak melakukan kecurangan seperti yang 
# telah dispesifikasikan. Aamiin.

# ----- Latihan Python ----- #

# Membuat kelas Product
class Product:
    # Membuat beberapa atirbut private
    # id Product
    __id = ""
    # name Product
    __name = ""
    # brand Product
    __brand = ""
    # price Product
    __price = ""

    # Constructor with default initialization
    def __init__(self, id="", name="", brand="", price=""):
        self.__id = id
        self.__name = name
        self.__brand = brand
        self.__price = price

    # Setter and Getter methods for private attributes
    # set ID
    def setID(self, id):
        self.__id = id

    # Get ID
    def getID(self):
        return self.__id

    # Set Name
    def setName(self, name):
        self.__name = name

    # Get Name
    def getName(self):
        return self.__name

    # Set Gender
    def setBrand(self, brand):
        self.__brand = brand

    # Get Gender
    def getBrand(self):
        return self.__brand

    # Set Sleeve Type
    def setPrice(self, price):
        self.__price = price

    # Get Sleeve Type
    def getPrice(self):
        return self.__price

File: PHP/Product.php
<!-- 
    Saya Alif Faturahman Firdaus (2107377) mengerjakan Latihan Praktikum 2 dalam mata 
    kuliah DPBO untuk keberkahan-Nya maka saya tidak melakukan kecurangan seperti yang 
    telah dispesifikasikan. Aamiin. 
-->

<!-- ----- Latihan PHP ----- -->

<?php
class Product
{
    // Membuat atribut Private dari kelas Product
    private $id = '';
    private $name = '';
    private $brand = '';
    private $price = '';
    private $image;

    # Constructor with default initialization
    public function __construct($id = '', $name = '', $brand = '', $price = '',  $image)
    {
        $this->id = $id;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
        $this->image = $image;
    }

    # Setter and Getter methods for private attributes

    # Set ID
    public function setID($idMahasiswa)
    {
        $this->id = $idMahasiswa;
    }

    # Get ID
    public function getID()
    {
        return $this->id;
    }

    # Set Name
    public function setName($name)
    {
        $this->name = $name;
    }

    # Get Name
    public function getName()
    {
        return $this->name;
    }

    # Set Brand
    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    # Get Brand
    public function getBrand()
    {
        return $this->brand;
    }

    # Set Price
    public function setPrice($price)
    {
        $this->price = $price;
    }

    # Get Price
    public function getPrice()
    {
        return $this->price;
    }

    # Set Image
    public function setImage($image)
    {
        $this->image = $image;
    }

    # Get Image
    public function getImage()
    {
        return $this->image;
    }
}